test/unit_tests/apps/owning-a-home/js/explore-rates/rate-checker-spec.js

```diff
@@ -19,7 +19,21 @@ const HTML_SNIPPET = `
         </table>
       </div>
 
-      <section id="chart-section">
+      <section id="chart-section" class="chart">
+
+        <figure class="data-enabled loading">
```

Comment:
Same question here about `loading` and on L:32 about indentation.

Reply:
It's okay!